.github/workflows/main.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11" ] # "3.12"
```

Comment:
Not working with py 3.12?

Reply:
There's a problem with `clisops` that prevents us from being able to install it in Python3.12. The last `xarray` version that rook supports is very old due to a regression that impacts `cftime` time encodings.

We would need to open this pin and deal with the problem somehow. Until then, my hands are tied.